app/shared/DynamicSVGIcon.js

```diff
@@ -0,0 +1,15 @@
+import InlineSVG from 'react-inlinesvg';
```

Comment:
Does this deserve to be in the shared folder?

Reply:
Um.. this is an `npm` package. You want me to bring it into our codebase? 🤔

Reply:
Oh, you're talking about the whole file, not the line this comment is referencing.  

Yeah I dunno.  I'm kinda agnostic to wherever we'd like it to live.  I figured since it's being used by the `SVGIconChooser`, and that's being used by tow other components, that it just made sense to be, to put it into something called "shared". 🤷 

But yeah, I guess maybe not? I'm ok w/ moving it wherever.